test/morpho-blue/TestBlueSnippets.sol

```diff
@@ -108,19 +108,19 @@ contract TestIntegrationSnippets is BaseTest {
     }
 
     function testBorrowAPY(uint256 amountSupplied, uint256 amountBorrowed, uint256 timeElapsed, uint256 fee) public {
-        vm.assume(fee < 1 ether);
+        fee = bound(fee, 1, MAX_FEE);
 
         _generatePendingInterest(amountSupplied, amountBorrowed, timeElapsed, fee);
         morpho.accrueInterest(marketParams);
 
         Market memory market = morpho.market(id);
-        uint256 borrowApyTrue = irm.borrowRate(marketParams, market).wTaylorCompounded(365 days);
-        uint256 borrowApyTest = snippets.borrowAPY(marketParams, market);
+        uint256 expectedBorrowApy = irm.borrowRate(marketParams, market).wTaylorCompounded(365 days);
+        uint256 borrowApy = snippets.borrowAPY(marketParams, market);
 
-        assertEq(borrowApyTrue, borrowApyTest, "Diff in snippets vs integration borrowAPY test");
+        assertEq(expectedBorrowApy, borrowApy, "Diff in snippets vs integration borrowAPY test");
```

Comment:
expectedBorrowApy and borrowApy should be inversed. The tested variable should be the first one